sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -507,6 +507,10 @@ public void populateDisplayData(DisplayData.Builder builder) {
                   DisplayDataMatchers.hasValue(fn.getClass().getName()))));
 
       assertThat(displayData, includesDisplayDataFor("fn", fn));
+
+      // Test setting DisplayData through PTransform API
+      parDo.setDisplayData(ImmutableList.of(DisplayData.item("doFnMetadata", "baz")));
```

Comment:
This test is mostly redundant with `PTransformTest`, I was just checking that it works with indirect methods of PTransform construction